charts/trino/values.yaml

```diff
@@ -218,6 +218,13 @@ securityContext:
   runAsUser: 1000
   runAsGroup: 1000
 
+# -- SecurityContext configuration for containers
+containerSecurityContext:
+  allowPrivilegeEscalation: false  #  Controls whether a process can gain more privileges than its parent process.
+  capabilities:
+    drop:  # List of Linux kernel capabilities that are dropped from every container. You can confirm the options for "capabilities" here:　https://man7.org/linux/man-pages/man7/capabilities.7.html  Please make sure to remove "CAP_" prefix which the kernel attaches to the names of permissions.
+      - ALL
```

Comment:
Is this meaningful and good default value? Is it compatible with the current behavior?

Reply:
yes, because the Trino containers will not require any Linux kernel capabilities.
It would be a best practice to remove unnecessary capabilities from containers.
I've confirmed that this works fine in our environment.

Reply:
Thanks for confirming